pkg/ali/l10n/en-us.yaml

```diff
@@ -11,4 +11,123 @@ ali:
     label: Cluster to import
 cluster:
   provider:
-    alibaba: Alibaba Cloud
\ No newline at end of file
+    alibaba: Alibaba Cloud
+ack:
+  location:
+    label: Location
+  accordions:
+    cluster: Cluster
+    basics: Basics
+    networking: Networking
+    labels: Labels and Annotations
+    advanced: Advanced
+  nodePool:
+    name: 
+      label: Node Pool Name
+    unnamed: Unnamed Node Pool
+    desiredSize:
+      label: Expected Nodes
```

Comment:
/nit not sure about calling this "expected", is the number of pools static? (Guessing yes because no autoscaling inputs). I think just calling this node count would be more consistent with the other cluster provisioners